### PR TITLE
LPS-97199 Apply changes

### PR DESCRIPTION
**Pull request by darquesdev**

### CODE REVIEW

**Comment by darquesdev** [comment]
ci:test:relevant

**Comment by darquesdev** [comment]
ci:test:sf

File: modules/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/client/model/util/ExperimentUtil.java [hunk 1]
```diff
@@ -38,26 +38,35 @@
...
+			_getLayoutFullURL(
+				portal, companyLocalService, groupLocalService,
+				layoutLocalService.getLayout(segmentsExperiment.getClassPK())),
+			segmentsEntryLocalService, segmentsExperienceLocalService,
+			segmentsExperiment);
 	}
 
-	public Experiment toDTO(
-			SegmentsExperiment segmentsExperiment, String dataSourceId)
+	protected static Experiment toExperiment(
```
Comment (darquesdev) [inline modules/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/client/model/util/ExperimentUtil.java]: @bianchandotcom I've added this method to facilitate unit test implementation which I'll send in a further PR

**Comment by epgarcia** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**